bin/node/src/service.rs

```diff
@@ -207,64 +178,44 @@ pub fn new_full(mut config: Configuration) -> Result<TaskManager, ServiceError>
         })
     };
 
-    let _rpc_handlers = sc_service::spawn_tasks(sc_service::SpawnTasksParams {
-        network: network.clone(),
-        client: client.clone(),
-        keystore: keystore_container.sync_keystore(),
-        task_manager: &mut task_manager,
-        transaction_pool: transaction_pool.clone(),
-        rpc_extensions_builder,
-        on_demand: None,
-        remote_blockchain: None,
-        backend,
-        system_rpc_tx,
-        config,
-        telemetry: telemetry.as_mut(),
-    })?;
-
+    let (_rpc_handlers, _telemetry_connection_notifier) =
```

Comment:
I'm a bit confused – telemetry has been removed as a crate, but there is something here. Do we have telemetry? Have we lost it? Did we ever have or use it?

Reply:
Right now, we have only support for prometheus. If we wanted telemetry, some work would have to be done.